rest/config.go

```diff
@@ -27,9 +27,8 @@ import (
 	"strings"
 	"syscall"
 
-	"golang.org/x/exp/maps"
-
 	"golang.org/x/crypto/bcrypt"
+	"golang.org/x/exp/maps"
```

Comment:
I don't think we want to import anything from exp - there's no API compatibility guarantees in the experimental repo.